po/tr.po

```diff
@@ -0,0 +1,706 @@
+msgstr "root olarak paket kurulamadı"
+
+#: src/install.rs:142
+msgid "failed to parse srcinfo generated by makepkg"
+msgstr "makepkg tarafından oluşturulmuş srcinfo çekilemedi"
+
+#: src/install.rs:171 src/install.rs:442
+msgid "Resolving dependencies..."
+msgstr "Gereksinimler çözülüyor..."
+
+#: src/install.rs:220 src/install.rs:225
+msgid "failed to download sources"
+msgstr "kaynaklar indirilemedi"
+
+#: src/install.rs:230 src/install.rs:246 src/install.rs:254
+msgid "failed to build"
+msgstr "kurulamadı"
+
+#: src/install.rs:233
+msgid "parsing pkg list..."
+msgstr "pkg listesi çekiliyor..."
+
+#: src/install.rs:261 src/install.rs:1462
+msgid "{}-{} is up to date -- skipping build"
+msgstr "{}-{} güncel -- kurulum es geçiliyor"
+
+#: src/install.rs:333 src/query.rs:50 src/lib.rs:84
+msgid "error:"
+msgstr "hata:"
+
+#: src/install.rs:334
+msgid "could not get news"
+msgstr "haberler çekilemedi"
+
+#: src/install.rs:339 src/install.rs:521 src/install.rs:716 src/install.rs:848
+msgid "Proceed with installation?"
+msgstr "Yüklemeye devam et?"
+
+#: src/install.rs:356
+msgid "no targets specified (use -h for help)"
+msgstr "hedef belirtilmedi (-h kullanarak yardıma ulaşın)"
+
+#: src/install.rs:434 src/install.rs:461 src/search.rs:324 src/search.rs:360
+#: src/lib.rs:227
+msgid " there is nothing to do"
+msgstr " yapılacak hiç bir şey yok"
+
+#: src/install.rs:488
+msgid "can't install AUR package as root"
+msgstr "root olarak AUR paketi yüklenemez"
+
+#: src/install.rs:507
+msgid "Remove make dependencies after install?"
+msgstr "Yükleme sonrasında make gereksinimlerini sil?"
+
+#: src/install.rs:518
+msgid "Proceed to review?"
+msgstr "İncelemeye geç?"
+
+#: src/install.rs:559
+msgid "The following packages are not compatible with your architecture:"
+msgstr "Devamın gösterilen paketler mimariniz ile uyumlu değildir:"
+
+#: src/install.rs:573
+msgid "Would you like to try build them anyway?"
+msgstr "Yinede kurmayı denemek ister misiniz?"
+
+#: src/install.rs:662
+msgid "packages failed to build: {}"
+msgstr "kurulumu hata veren paketler: {}"
+
+#: src/install.rs:698
+msgid "could not find .SRINFO for '{}'"
+msgstr "'{}' için .SRINFO bulunamadı"
+
+#: src/install.rs:762
+msgid "failed to read dir: {}"
+msgstr "dizin okunamadı: {}"
+
+#: src/install.rs:778
+msgid ""
+"{} is a directory\n"
+"\n"
+msgstr ""
+"{} bir dizindir\n"
+"\n"
+
+#: src/install.rs:818
+msgid "failed to open: {}"
+msgstr "açılamadı: {}"
+
+#: src/install.rs:830
+msgid "binary file: {}"
+msgstr "ikili dosya: {}"
+
+#: src/install.rs:852
+msgid " nothing new to review"
+msgstr " inceleme için yeni bir şey yok"
+
+#: src/install.rs:871
+msgid "failed to execute file manager: {}"
+msgstr "dosya yöneticisi çalıştırılamadı: {}"
+
+#: src/install.rs:874
+msgid "file manager did not execute successfully"
+msgstr "dosya yöneticisi başarılı bir şekilde çalışmadı"
+
+#: src/install.rs:936
+msgid "duplicate packages: {}"
+msgstr "mükerrer paketler: {}"
```

Comment:
`mükerrer` is an oddly _old_ word to say in place of "duplicate". Please consider using `"Tekrar eden paketler"` since it is more understandable.

Reply:
@orhun thanks for the suggestions